### ISSUE TITLE
Error Parsing Quotation Marks in SQL Statement

### ISSUE THREAD

**Issue by HandyGuySoftware** [HandyGuySoftware/dupReport#126]
```
Database.execSqlStmt(): Executing SQL command.
SQL stmt=[INSERT INTO emails(messageId, sourceComp, destComp, emailTimestamp,             deletedFiles, deletedFolders, modifiedFiles, examinedFiles,             openedFiles, addedFiles, sizeOfModifiedFiles, sizeOfAddedFiles, sizeOfExaminedFiles,             sizeOfOpenedFiles, notProcessedFiles, addedFolders, tooLargeFiles, filesWithError,             modifiedFolders, modifiedSymlinks, addedSymlinks, deletedSymlinks, partialBackup,             dryRun, mainOperation, parsedResult, verboseOutput, verboseErrors, endTimestamp,             beginTimestamp, duration, messages, warnings, errors, dbSeen, dupversion, logdata)             VALUES             ('<1NVT81EUW8U4.3WEI9I0UE5IV3@Superman>', 'Media', 'Superman', 1572091721.0, 5, 0, 9, 3701, 75, 66, 32133419,149114617043,724893334774,149146766575,0,20,0,0,0,0,0,             '0', 'False', 'False', 'Backup', 'Warning', '', '', '1572091721.0', '1572091721.0', '0.0', "", "", "", 1, "2.0.4.30 (2.0.4.30_canary_2019-09-20)", "2019-10-26 06:55:03 -05 - [Warning-Duplicati.Library.Main.Operation.Backup.MetadataGenerator.Metadata-MetadataProcessFailed]: Failed to process metadata for "N:\"
 storing empty metadata
System.InvalidOperationException: Method failed with unexpected error code 21.
   at System.Security.AccessControl.NativeObjectSecurity.CreateInternal(ResourceType resourceType
 Boolean isContainer
 String name
 SafeHandle handle
 AccessControlSections includeSections
 Boolean createByName
 ExceptionFromErrorCode exceptionFromErrorCode
 Object exceptionContext)
   at System.Security.AccessControl.FileSystemSecurity..ctor(Boolean isContainer
 String name
 AccessControlSections includeSections
 Boolean isDirectory)
   at System.Security.AccessControl.DirectorySecurity..ctor(String name
 AccessControlSections includeSections)
   at System.IO.Directory.GetAccessControl(String path)
   at Duplicati.Library.Common.IO.SystemIOWindows.PathTooLongFuncWrapper[T](Func`2 nativeIOFunc
 Func`2 alternativeIOFunc
 String path
 Boolean prefixWithUnc)
   at Duplicati.Library.Common.IO.SystemIOWindows.GetAccessControlDir(String path)
   at Duplicati.Library.Common.IO.SystemIOWindows.GetMetadata(String path
 Boolean isSymlink
 Boolean followSymlink)
   at Duplicati.Library.Main.Operation.Backup.MetadataGenerator.GenerateMetadata(String path
 FileAttributes attributes
 Options options
 ISnapshotService snapshot)")]
SQLite error: near "N": syntax error

```

**Comment by HandyGuySoftware** [comment]
Fixed in 2.2.9

**Comment by HandyGuySoftware** [comment]
Yeah, so this problem still exists. Need to find a way to escape out single & double quotes in SQL statements.

**Comment by nahall** [comment]
I encountered this bug and tried your Issue_126 branch, and that seems to be working.

**Comment by HandyGuySoftware** [comment]
Thanks, @nahall. Since this is the second time around with this same bug I've been letting this one burn in a few weeks longer than normal. Let yours run for another week or so and if neither of us encounters the issue again I'll promote this directly up to Master.

**Comment by nahall** [comment]
> Thanks, @nahall. Since this is the second time around with this same bug I've been letting this one burn in a few weeks longer than normal. Let yours run for another week or so and if neither of us encounters the issue again I'll promote this directly up to Master.

Sounds good, thanks!

**Comment by HandyGuySoftware** [comment]
Fixed (again) in release 2.2.10.
